### PR TITLE
Add test id to VirtOperatorRESTErrorsHigh

### PR DESCRIPTION
**Pull request by avlitman**
**What this PR does / why we need it**:
Adding marking to identify the test on execution for VirtOperatorRESTErrorsHigh test.
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```
NONE

```

### CODE REVIEW

**Comment by machadovilaca** [comment]
@avlitman please add 'NONE' to the release note

**Comment by avlitman** [comment]
> @avlitman please add 'NONE' to the release note

Added, thanks!!

File: tests/monitoring/component_monitoring.go [hunk 1]
```diff
@@ -195,7 +195,7 @@ var _ = Describe("[Serial][sig-monitoring]Component Monitoring", Serial, decorat
 			waitUntilAlertDoesNotExist(virtClient, virtOperator.downAlert, virtApi.downAlert, virtController.downAlert, virtHandler.downAlert)
 		})
 
-		It("VirtApiRESTErrorsBurst and VirtApiRESTErrorsHigh should be triggered when requests to virt-api are failing", func() {
+		It("[test_id:9995] VirtApiRESTErrorsBurst and VirtApiRESTErrorsHigh should be triggered when requests to virt-api are failing", func() {
```
Comment (fossedihelm) [inline tests/monitoring/component_monitoring.go:198]: `VirtOperatorRESTErrorsHigh` != `VirtApiRESTErrorsHigh`
Wrong test?

Comment (avlitman) [reply tests/monitoring/component_monitoring.go:198]: No, I  asked QE and it's this one. `VirtApiRESTErrorsBurst and VirtApiRESTErrorsHigh` I don't see any other test with `VirtApiRESTErrorsHigh`

Comment (avlitman) [reply tests/monitoring/component_monitoring.go:198]: I talked with Ohad Revah

**Comment by EdDev** [comment]
> As QE asked, adding the polarion id for VirtOperatorRESTErrorsHigh test.

I know this was the norm for some time now, but I think this is wrong.
First, we should no longer use tags but labels for marking tests.
Then, one needs to give a much better reasoning to add a marking like this, especially when the project is not using it AFAIK.

**Comment by avlitman** [comment]
I'm not sure it's my decision to make, but can you show me how to make it as you want?

**Comment by EdDev** [comment]
> I'm not sure it's my decision to make, but can you show me how to make it as you want?

If you are asking about labels, then: https://onsi.github.io/ginkgo/#spec-labels
It obviously needs to be in sync with the one who consumes it.

File: tests/monitoring/component_monitoring.go (hunk 1 above)
Comment (avlitman) [reply tests/monitoring/component_monitoring.go:198]: Soryy my bad miss the API, fixing.

**Comment by avlitman** [comment]
I rebased hope it helps @fossedihelm

**Comment by avlitman** [comment]
can we approve this? @fossedihelm

**Comment by avlitman** [comment]
@dhiller @fossedihelm I missed that this pr is still open, can we approve this?

**Pull request closed without merging**